docs/steps.md

```diff
@@ -0,0 +1,68 @@
+### Node Setup
+
+```bash
+sudo apt update
+sudo apt install -y jq
+jq --version
+```
+
+```bash
+git clone https://github.com/zeta-chain/network-athens3.git && cd network-athens3
+```
+
+- Branch name will be provided by the coordinator
```

Comment:
Who is the coordinator? Looking at this from the outside operator perspective they won't have any context for who/what the coordinator is.  

I think changing this to  "zetachain coordinator" will make it a little clearer to outside operators